emcc.py

```diff
@@ -1246,8 +1272,11 @@ def get_bitcode_args(input_files):
             shared.Building.llvm_as(input_file, temp_file)
             temp_files.append((i, temp_file))
         else:
-          logging.error(input_file + ': Unknown file suffix when compiling to LLVM bitcode!')
-          sys.exit(1)
+          if(has_fixed_language_mode):
```

Comment:
I moved the check to here because normal source / bitcode / ... handling should be done by now, so I guess it's now save to do the compilation for source files without extension.